test_conformance/device_execution/enqueue_flags.cpp

```diff
@@ -185,7 +183,7 @@ static const char* enqueue_flags_wait_kernel_local[] =
     NL, "  size_t tid = get_global_id(0);"
     NL, ""
     NL, "  sub_array[lid] = array[(index - 1) * gs + tid];"
-    NL, "  barrier(CLK_LOCAL_MEM_FENCE);"
+    NL, "  barrier(CLK_GLOBAL_MEM_FENCE);"
```

Comment:
The local fence is needed here so all threads in the loop below see all the updates

Reply:
Ah, sorry. It's been a while, I completely forgot there were working barriers